tests/src/test/scala/com/twitter/intellij/pants/OpenProjectTest.scala

```diff
@@ -147,3 +138,19 @@ object OpenProjectTestPants extends OpenProjectTestFixture {
 object OpenProjectTestBsp extends OpenProjectTestFixture {
   override def openProject(): ProjectRef = openProjectWithBsp(intelliJ)
 }
+
+object OpenProjectTestFastpass extends OpenProjectTestFixture with ScalaPluginExtension {
+  private def targetsFromConfig(intelliJ: RunningIntelliJFixture): Seq[String] =  {
+    intelliJ.config[Seq[String]]("pants.import.targets")
+  }
+
+  override def openProject(): ProjectRef = {
+    val path =  intelliJ.workspace.resolve(targetsFromConfig(intelliJ).head)
+    val project = intelliJ.probe.importBspProject(path)
+    project
+  }
+}
+
+class OpenProjectTestFastpass extends CommonOpenProjectTests {
```

Comment:
You may want to reuse tests a bit differently here. Before this PR we had 2 things:
Bsp and Pants. For these we had Common trait. Each of these has additional custom tests. I think what should be done now is to have: Common and Bsp base traits and BspConsole and BspUi that inherit from Bsp. Currently `OpenProjectTestFastpass ` is not testing vcs setup for example.